bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -363,8 +363,8 @@ export class BoxAnnotationView extends AnnotationView implements Pannable, Pinch
       ctx.rect(x, y, width, height)
       round_rect(ctx, this.bbox, this.border_radius)
 
-      fill.apply(ctx, "evenodd")
-      hatch.apply(ctx, "evenodd")
+      fill.apply(ctx, undefined, "evenodd")
+      hatch.apply(ctx, undefined, "evenodd")
```

Comment:
Is this necessary? Do devs need to take care to pass an undefined `path` everywhere now? If it it not necessary, I'd suggest omitting it to avoid confusion. If it is necessary, I'd suggest reworking the API so that it is not necessary.

Reply:
I improved the API. No need for `undefined` anymore.